.kokoro/build.bat

```diff
@@ -1,3 +1,3 @@
 :: See documentation in type-shell-output.bat
 
-"C:\Program Files\Git\bin\bash.exe" github/java-storage/.kokoro/build.sh
+"C:\Program Files\Git\bin\bash.exe" %~dp0build.sh
```

Comment:
Is this an expected changed?

Reply:
The kokoro changes are a result of the old branch (mar 2020) being out of date with the latest config